src/DataGrid.tsx

```diff
@@ -1061,7 +1065,6 @@ function DataGrid<R, SR, K extends Key>(
   // Reset the positions if the current values are no longer valid. This can happen if a column or row is removed
   if (selectedPosition.idx > maxColIdx || selectedPosition.rowIdx > maxRowIdx) {
     setSelectedPosition({ idx: -1, rowIdx: minRowIdx - 1, mode: 'SELECT' });
-    setDraggedOverRowIdx(undefined);
```

Comment:
Is this needed? We do set it to undefined after the drag operation 
https://github.com/adazzle/react-data-grid/blob/main/src/DragHandle.tsx#L103

Reply:
Probably. I added it back for now. I will look into alternatives